SwiftWisdom/Core/NSFormatter/DecimalFormatter.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DecimalFormatter.swift
+//  SwiftWisdom
+//
+//  Created by Stephen Wong on 5/1/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+final class DecimalFormatter: NSNumberFormatter {
```

Comment:
im not sure I agree with creating a top level subclass called DecimalFormatter here - after all this does not add any new functionality to NSNumberFormatter. This looks like a convenience initializer or an extension that returns a nsnumberformatter, at most.